web/src/locales/en.json

```diff
@@ -124,14 +124,16 @@
       "primary-question": "The description of the data estimation was easy to understand:"
     },
 
-    "primary-question": "How satisfied are you with your experience using the Electricity Maps app?",
+    "primary-question": "How would you rate your overall satisfaction with the Electricity Maps app?",
```

Comment:
Is this added to any other language that we need to adjust as well?

Reply:
Yes it's also in 3 other languages which translate to "How satisfied are you with your experience using the Electricity Maps app?" I think it's a finer detail though, here's the Swedish: "Hur nöjd är du med din upplevelse av att använda Electricity Maps-appen?" Does it still capture the general question?

Reply:
Yeah I think that would still work. I can add the Swedish translations for the new strings if you wish?

Reply:
yeah sounds good!